README.md

```diff
@@ -86,6 +86,7 @@ Visit this documentation page https://www.open-csp.org/DevOps:Doc/FlexForm/2.0/V
 Visit : https://www.open-csp.org/DevOps:Doc/FlexForm
 
 ### Changelog
+* 2.6.3 : Changed Debug mode. Fixed leadingzero function. $output not defined fixed. Fix create range when it cannot be set. Have MW check page before it is created.
```

Comment:
I'd still prefer if you'd give some more details on what is changed about debug mode, but I'll let it pass for now.